build/gulpfile.vscode.linux.js

```diff
@@ -142,6 +175,76 @@ function buildRpmPackage(arch) {
 	]);
 }
 
+function getFlatpakArch(arch) {
+	return { x64: 'x86_64', ia32: 'i386', arm: 'arm' }[arch];
+}
+
+function prepareFlatpak(arch) {
+	// This is not imported in the global scope to avoid requiring ImageMagick
+	// (or GraphicsMagick) when building not building Flatpak bundles.
+	const imgResize = require('gulp-image-resize');
+
+	const binaryDir = '../VSCode-linux-' + arch;
+	const flatpakArch = getFlatpakArch(arch);
+	const destination = '.build/linux/flatpak/' + flatpakArch;
+
+	return function () {
+		const all = [16, 24, 32, 48, 64, 128, 192, 256, 512].map(function (size) {
```

Comment:
Are all these sizes necessary? The smaller ones in particular would probably look pretty crappy being scaled like this?